easybuild/easyconfigs/s/scikit-image/scikit-image-0.17.2-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-image'
+version = '0.17.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://scikit-image.org/'
+description = "scikit-image is a collection of algorithms for image processing."
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('matplotlib', '3.2.1', versionsuffix),
+    ('Pillow', '7.0.0', versionsuffix),
+    ('dask', '2.18.1', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('tifffile', '2020.7.4', {
+        'checksums': ['d7e56b9983e2e14716feaee560022baa8f92e97f36df7eee8b10e672383f694c'],
+    }),
+    ('networkx', '2.4', {
+        'checksums': ['f8f4ff0b6f96e4f9b16af6b84622597b5334bf9cae8cf9b2e42e7985d5c95c64'],
+    }),
+    ('PyWavelets', '1.1.1', {
+        'modulename': 'pywt',
+        'checksums': ['1a64b40f6acb4ffbaccce0545d7fc641744f95351f62e4c6aaa40549326008c9'],
+    }),
+    ('imageio', '2.6.1', {
```

Comment:
@bjmoran The existing easyconfig for scikit-image 0.17.1 uses `imageio` 2.9.0, is there a reason you are downgrading that here?

Same question for `imread` (existing easyconfig has `imread` 0.7.4)

Reply:
That scikit-image 0.17.1 easyconfig is newer than this pull request. I'll have a look at it.